docker-compose.yml

```
@@ -27,3 +27,13 @@ services:
       - '1323:1323'
     tty: true
     stdin_open: true
+  view:
+    build: ./view
+    container_name: 'view'
+    ports:
+      - "3000:3000"
```

```suggestion
    stdin_open: true
    
  view:
    build: ./view
    container_name: 'view'
    ports:
      - "3000:3000"
      - "6006:6006"
```
30行目は改行あった方が見やすいと思います！
あと`storybook`を開く際にportの6006番を利用していると思うので、6006番を解放してあげないと`storybook`は開けないかと思います。(自分は6006を解放したら開けました)